pywinauto/uia_defines.py

```diff
@@ -168,11 +168,16 @@ class NoPatternInterfaceError(Exception):
     pass
 
 # values for enumeration 'ExpandCollapseState'
-expand_state_collapsed =IUIA().ui_automation_client.ExpandCollapseState_Collapsed
+expand_state_collapsed = IUIA().ui_automation_client.ExpandCollapseState_Collapsed
 expand_state_expanded = IUIA().ui_automation_client.ExpandCollapseState_Expanded
 expand_state_partially = IUIA().ui_automation_client.ExpandCollapseState_PartiallyExpanded
 expand_state_leaf_node = IUIA().ui_automation_client.ExpandCollapseState_LeafNode
 
+# values for enumeration 'WindowVisualState'
+window_visual_state_normal = IUIA().ui_automation_client.WindowVisualState_Normal
+window_visual_state_maximized = IUIA().ui_automation_client.WindowVisualState_Maximized
+window_visual_state_minimized = IUIA().ui_automation_client.WindowVisualState_Minimized
```

Comment:
It looks like simple enumeration. Maybe not now, but we may think about something like that:
```python
class WindowVisualState(object):
    normal = IUIA().ui_automation_client.WindowVisualState_Normal
    maximized = IUIA().ui_automation_client.WindowVisualState_Maximized
    minimized = IUIA().ui_automation_client.WindowVisualState_Minimized

# and then use WindowVisualState.normal etc.
```